client/src/components/categorical/index.tsx

```diff
@@ -133,6 +133,41 @@ class Categories extends React.Component<{}, State> {
     }
   };
 
+  /**
+   * Determine if category name is an ontology key.
+   * @param catName - Name of category.
+   * @returns True if category name includes ontology key.
+   */
+  isCategoryNameOntologyKey = (catName: string): boolean =>
+    catName.includes(globals.ONTOLOGY_KEY);
+
+  /**
+   * Categories are included for display if category has more than one category value or categories are not defined
+   * (for the case where category is a string or boolean type).
+   * @param schema - Matrix schema.
+   * @param catName - Name of category.
+   * @returns True if category has more than one category value or categories are not defined.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types -- update on typing of component
+  isCategoryDisplayable = (schema: any, catName: string): boolean =>
+    schema.annotations.obsByName[catName].categories?.length > 1 ||
+    !schema.annotations.obsByName[catName].categories;
```

Comment:
We can probably change this line since we have typing to make it more readable!

Maybe something like 
```suggestion
    !schema.annotations.obsByName[catName] instanceof CategoricalAnnotationColumnSchema
```

Reply:
I am getting a `TS2693: 'CategoricalAnnotationColumnSchema' only refers to a type, but is being used as a value here.` error using `instanceof`.

I have updated the code to use the `in` operator to narrow the type of `schema.annotations.obsByName[catName]` to `CategoricalAnnotationColumnSchema` when checking the `categories` length but I had trouble getting it any clearer than that. I'm open to suggestions on improving readability here!

Reply:
Hmm yeah I think if we really wanted to go one step further, we'd need to create a helper function like: 

```ts
function isCategoricalAnnotationColumnSchema(schema: AnnotationColumnSchema): schema is CategoricalAnnotationColumnSchema {
  if (("categories" in columnSchema)) {
      return true;
    }

  return false;
}
```

But I don't feel strongly about it, since TS already narrows the type like Mim says 👌 

And it makes sense that `!schema.annotations.obsByName[catName] instanceof CategoricalAnnotationColumnSchema` doesn't work, since `instanceof` only works on `classes` and `CategoricalAnnotationColumnSchema` is an interface?